docs/CONTRIBUTING.md

```diff
@@ -9,6 +9,36 @@ Hi there! We're thrilled that you'd like to contribute to this project. Your hel
 
 Please note that this project is released with a [Contributor Code of Conduct][code-of-conduct]. By participating in this project you agree to abide by its terms.
 
+## Development setup & configuration
+
+After forking the repository you can deploy the project service to your own Heroku backend environments for production and staging. To be able to do so, you must first create the prerequisite environments using a Heroku account and login through the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) using the following steps.
+
+A) Use the following command to create a staging environment:
+
+`heroku create --remote staging`
+
+Note the name of the app created e.g `git.heroku.com/strong-river-216.git`, this is the Heroku staging environment name.
```

Comment:
It's semantics and subjective but the thing you're referring to here is the Heroku git repository for the staging app/environment. I'm not 100% sure about Heroku terminology but I think `strong-river-216` is the app name and `git.heroku.com/strong-river-216.git` the repository name.